ddl/ddl_worker.go

```diff
@@ -149,7 +147,20 @@ func asyncNotify(ch chan struct{}) {
 // buildJobDependence sets the curjob's dependency-ID.
 // The dependency-job's ID must less than the current job's ID, and we need the largest one in the list.
 func buildJobDependence(t *meta.Meta, curJob *model.Job) error {
-	jobs, err := t.GetAllDDLJobs()
+	// Jobs in the same queue are ordered. If we want to find a job's dependency-job, we need to look for
+	// it from another queue. So if the job is "ActionAddIndex" job, we need find its dependency-job from DefaultJobList.
+	// And after looking for dependency-job, we need to change the queue to the queue corresponding to the current job.
```

Comment:
Can not understand "we need to change the queue to the queue corresponding to the current job".